cluster/sync.sh

```diff
@@ -77,5 +101,34 @@ for i in {300..0}; do
     exit 1
     fi
 
-    sleep 1
-done
+
+    # Deploy all needed manifests
```

Comment:
This is now duplicated on line 76

Reply:
I am blind, where it is duplicated ?

Reply:
https://github.com/nmstate/kubernetes-nmstate/pull/448/files/75639d9aa18d845b776fa6416c5908ae05e81280#diff-9a65a9e4d7562183f4e925310683fa9fR76